### PR TITLE
spec for new element unit and Guidelines explanation for #1461

### PR DESCRIPTION
**Pull request by ebeshero**
@martinascholger @jamescummings @emylonas Here's a spec for the new element `<unit>` that seems timely for our upcoming release. Can you review this and see if it's okay to merge into dev?

### CODE REVIEW

**Comment by jamescummings** [comment]
I don't see anything broken giving it a quick glance on the metro home.

**Comment by martinascholger** [comment]
The example in the GL is in French "millimètres". 
If you want to add a German translation, "enthält ein Symbol, ein Wort oder eine Phrase, die sich auf eine Maßeinheit in einem formellen oder informellen System bezieht."

File: P5/Source/Guidelines/en/CO-CoreElements.xml [hunk 1]
```diff
@@ -1935,6 +1935,12 @@ avoid the need to repeat all the normalizing attributes:
     <measure type="list"><num>1</num>, <num>2</num>, <num>5</num>, <num>7</num>
       <unit type="length" norm="mm">millimètres</unit></measure>
```
Comment (martinascholger) [inline P5/Source/Guidelines/en/CO-CoreElements.xml]: the word "millimètres" is still in French (and it says @xml:lang="en" in `egXML`)

**Comment by martinascholger** [comment]
For the second example the xml:lang="en" is missing

**Comment by ebeshero** [comment]
@martinascholger Oops--thanks for catching that. Take a look now.

**Comment by ebeshero** [comment]
By the way, I'm not certain of the status / usage of the `@source` attribute on `<egXML>` here, but I took my cue from other examples in the Guidelines. The examples included here are from the ticket and there are more on the spec page.

**Comment by martinascholger** [comment]
Looks good now.

**Comment by ebeshero** [comment]
I'm adding a sentence with a little more explanation/suggestion for usage...

**Comment by ebeshero** [comment]
Okay to merge this into `dev` and resolve #1461 for now?

**Comment by ebeshero** [comment]
(As long as Jenkins agrees--it looks peaceful now.)

**Comment by ebeshero** [comment]
I'm just going to merge this, since @martinascholger , @jamescummings and I reviewed it and we're green for resolving ticket #1461 .

**Pull request merged**